src/lib/constants/dev-environment.ts

```diff
@@ -28,11 +28,29 @@ export const DEV_ENVIRONMENT_WORDPRESS_CACHE_KEY = 'wordpress-versions.json';
 
 export const DEV_ENVIRONMENT_WORDPRESS_VERSION_TTL = 86400; // once per day
 
-export const DEV_ENVIRONMENT_PHP_VERSIONS: Record< string, string | undefined > = {
-	'8.0': 'ghcr.io/automattic/vip-container-images/php-fpm:8.0',
-	8.2: 'ghcr.io/automattic/vip-container-images/php-fpm:8.2',
-	8.1: 'ghcr.io/automattic/vip-container-images/php-fpm:8.1',
-	7.4: 'ghcr.io/automattic/vip-container-images/php-fpm:7.4',
+interface PhpImage {
+	image: string;
+	label: string;
+}
+
+export const DEV_ENVIRONMENT_PHP_VERSIONS: Record< string, PhpImage > = {
+	8.1: { image: 'ghcr.io/automattic/vip-container-images/php-fpm:8.1', label: '8.1 (recommended)' },
+	8.2: {
+		image: 'ghcr.io/automattic/vip-container-images/php-fpm:8.2',
```

Comment:
We should mark 8.2 as recommended at this point now given that we're moving towards everyone being on it now, yeah?

Reply:
Done.